src/containers/TasksPage/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import ReactDOM from "react-dom";
+import styled from 'styled-components'
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import bootstrapPlugin from "@fullcalendar/bootstrap";
+import interactionPlugin from "@fullcalendar/interaction";
+import MediaQuery from 'react-responsive'
+
+import "@fullcalendar/core/main.css";
+import "@fullcalendar/daygrid/main.css";
+import "@fullcalendar/bootstrap/main.css";
+import "./dashbroad.css";
+
+export default class DashBroad extends Component {
+  constructor(props) {
+    super(props);
+
+    const events = [
+        {
+            title: '5',
+            description: 'Tasks',
+            start: '2019-07-01',
+            color: 'red',
+        },
+        {
+            title: '10',
+            description: 'Tasks',
+            start: '2019-07-01',
+        },
+        {
+            title: '10',
+            description: 'Tasks',
+            start: '2019-07-19T16:00:00',
+        },
+    ];
+    this.state = {
+      events,
+    };
+  }
+  
+  render() {
+    const events = this.state
+    return (
+      <div id="calendar" className="container" ref="calendar">
+        <FullCalendar
+          selectable={true}
```

Comment:
```suggestion
          selectable
```
ko pass boolean thô như này a nhá, viết ko như thế này nó hiểu là `selectable=true`, ko viết thì tức là false :v